metadata/algolia/meta/system-objecttype-extensions.xml

```diff
@@ -123,7 +123,7 @@
                 <display-name xml:lang="x-default">Search API key</display-name>
                 <description xml:lang="x-default">This is the public API key which can be safely used in your frontend code.This key is usable for search queries and it's also able to list the indices you've got access to.</description>
                 <type>string</type>
-                <mandatory-flag>false</mandatory-flag>
+                <mandatory-flag>true</mandatory-flag>
```

Comment:
Actually maybe this one shouldn't be mandatory, as people with headless implementations won't need it.

Reply:
Agreed, this would create a problem for these clients.

Reply:
Reverted